docker/Dockerfile.aarch64-unknown-linux-gnu

```diff
@@ -12,10 +12,11 @@ RUN /xargo.sh
 
 FROM cross-base as build
 
-RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
-    g++-aarch64-linux-gnu \
-    gfortran-aarch64-linux-gnu \
-    libc6-dev-arm64-cross
+ENV CROSS_TOOLCHAIN_PREFIX=aarch64-linux-gnu-
+ENV CROSS_SYSROOT=/usr/aarch64-linux-gnu
+
+COPY apt-cross-essential.sh /
+RUN TARGET_ARCH=arm64 TARGET_TRIPLE=aarch64-linux-gnu /apt-cross-essential.sh
```

Comment:
Hmm, looking at this again makes me suspicious of this approach, it makes it harder to debug with a glance whats extra in the image. Id prefer it if the script was removed or if the wanted packages were explicitly passed as arguments

Reply:
reverted these changes

Reply:
I think there will be some problems with this, due to what was [mentioned here](https://github.com/cross-rs/cross/pull/1597#discussion_r1889656345)